### ISSUE TITLE
Ignoring minItems constraint while building json schema

### ISSUE THREAD

**Issue by SweetieRapuncel** [victools/jsonschema-generator#15]
I tried to build json schema using jsonschema-module-javax-validation module and found that minItems constraint is ignored. Also i found that jsonschema-module-javax-validation module works as expected and com.github.victools.jsonschema.generator.impl.AttributeCollector builds node with expected constraints, but com.github.victools.jsonschema.generator.SchemaGenerator ignores them.

May be i have erroneous configuration)
![json-schema](https://user-images.githubusercontent.com/40152112/71573981-9dbe3b00-2af7-11ea-8241-f02663e18564.png)

**Comment by CarstenWickner** [comment]
Hi @SweetieRapuncel,

Could you please share what your configuration is?
I.e. how the `SchemaGenerator` is instantiated and configured.
Maybe the `minItems` are overridden by something else.

**Comment by SweetieRapuncel** [comment]
Hello @CarstenWickner (:
I use this code to instantiate SchemaGenerator: 
`new SchemaGenerator(new SchemaGeneratorConfigBuilder(new ObjectMapper(), OptionPreset.PLAIN_JSON).with(new JavaxValidationModule()).build())`

**Comment by CarstenWickner** [comment]
Hi @SweetieRapuncel,

Thanks for reporting this.
I can reproduce it and can confirm that there is definitely a bug somewhere. Investigating it right now.
Will keep you posted.

**Comment by CarstenWickner** [comment]
Hi @SweetieRapuncel,

I'm happy to let you know:
- the fix was quite straightforward
- the fix is included in release 3.4.1

Thanks again for reporting. 😄👍

**Comment by CarstenWickner** [comment]
Hi @SweetieRapuncel,

Thanks to your bug report I noticed another issue with the schema population of array items.
For these the "title", "description", etc. were sadly not even picked-up as they are strictly speaking not the declared type of a particular field/method.

I've fixed this omission now in #18 by introducing another set of configurations, released it as version 4.0.0 and updated the affected modules accordingly.
Now the quality of produced schemas involving array/collection types should be improved.

Just letting you know because that seemed close to your described use-case and I'm excited about the new capabilities. 😉
